utils/configen/mkconfig.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/keyserver"
```

Comment:
> Should I make it 2 separate utils?

I think you should move this stuff to,

```
coniksbot/main.go
coniksserver/main.go
```

which could each take a flag, `-genconfig` or something.